### ISSUE TITLE
Suppress "This build requires parameters" if all parameters are hidden

### ISSUE THREAD

**Issue by mdorotich-swi** [jenkinsci/hidden-parameter-plugin#137]
### What feature do you want to see added?

If a job's parameters are all hidden, is it possible to make it appear to the user as a job with no parameters?

- Trigger by pressing 'Build', not 'Build with Parameters'
- Bypass the (empty) 'This build requires parameters' dialog, and launch the build

**Comment by jonesbusy** [comment]
Hi,

Not sure if even possible. This plugin only use APIs from Jenkins core.

Or perhaps this deserve a better message if the build only have hidden parameters ?

I'll leave this open if someone is willing to research and check if possible

**Comment by chosroes** [comment]
I don't agree with that. I use this plugin for this feature. Building with parameters even if all parameters are hidden
